### ISSUE TITLE
Version says "Loading" and URL is releaser.richardhagen.no/

### ISSUE THREAD

**Issue by MrHubble** [Richard87/releaser#19]
Thanks for providing such a useful service. 

A number of the items in my watchlist have "Loading" as the version number and when I hover over that link the url is https://releaser.richardhagen.no/

![screen shot 2017-06-29 at 11 46 34 am](https://user-images.githubusercontent.com/2251631/27667880-18acac1e-5cc1-11e7-9edb-a0809d09dd7f.png)

**Comment by AlexWayfer** [comment]
I think, #5 can help you 😉

_Sent from my Sony E5823 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Richard87** [comment]
Hi, because releaser is closing in to the ceiling of the free data usage of FireBase, I had to scale back the refresh rate from my server, so it should update every 30minutes... Maybe I can update the text to something more accurate ;)

In other words, it should work now! :)

**Comment by AlexWayfer** [comment]
> Maybe I can update the text to something more accurate ;)

Maybe `Queued`? And without link, as `span`, for example.

**Comment by MrHubble** [comment]
I've been using releaser for a few weeks now and I think those same projects shown in the screenshot have always said "Loading". They have never updated to show the version number. Would this fit in with the 30 minute refresh rate? If yes, I wonder why some of the projects will show a version and then others wont.

**Comment by Richard87** [comment]
No, it would not, thanks for reporting it in, I'll look into it!

fre. 30. jun. 2017 kl. 00:45 skrev MrHubble <notifications@github.com>:

**Comment by Richard87** [comment]
Yes, please! :)

tor. 29. jun. 2017 kl. 14:25 skrev Alexander Popov <notifications@github.com

**Comment by bluefish6** [comment]
I also have the same issue - some repos don't show their version even after few days, please see the screenshot below:
![image](https://user-images.githubusercontent.com/3324881/28024112-19293f68-6590-11e7-9313-e5888a63d3e9.png)

**Comment by AlexWayfer** [comment]
The same :(

No notifications.

Previously worked properly…

![image](https://user-images.githubusercontent.com/6510020/28128141-7432b76a-6737-11e7-9325-77ddea0df43b.png)

**Comment by netx123** [comment]
Same issue here, I've tried this service for the first time today, adding three projects, but 4 hours later and they all still show as "Loading"

**Comment by Richard87** [comment]
Thanks guys, I'm on vacation right now, but I'll look into it asap when I
get back in 2 weeks!

Richard

**Comment by Richard87** [comment]
Small update on this issue: Someone have registered with the email `firstname at lastname dot cc`, which crashed my cron-job, this issue *might* be solved, atleast it works for `raven-python` and `uwsgi`

On a side note, my plan is to migrate all my code to check for updates to firebase-functions, so all code should be in this repo, and if anyone wants to help out, I would love it!

**Comment by AlexWayfer** [comment]
`No releases` now instead of `Loading`.

And empty `href`.

What do you think about `https://github.com/%user_name%/%repo_name%/releases` URL for not-loaded links?

**Comment by Richard87** [comment]
Which repo was that?

**Comment by AlexWayfer** [comment]
@Richard87 Oh, I don't remember, sorry. Any freshly-added repo, I guess. And it was [zyedidia/micro](https://github.com/zyedidia/micro), I guess.

**Comment by AlexWayfer** [comment]
[`stylelint/stylelint`](https://github.com/stylelint/stylelint) for now.
